src/element/shape.js

```diff
@@ -594,43 +596,41 @@ export class ShapeElement extends Element {
   }
 
   drawShapePath(frame, shapePath, index) {
+    const beginProcess = pixiVersionHelper.select(() => {} /* for v4 API */, () => { this.beginHole(); } /* for v5 API */);
```

Comment:
This process will better move to constructor for performance . 
And I think about it will good to call as `this.beginProcess()`

Reply:
Fixed!